src/renderer/components/ReactFlowBox.tsx

```diff
@@ -673,9 +673,11 @@ export const ReactFlowBox = memo(({ wrapperRef, nodeTypes, edgeTypes }: ReactFlo
                 <Controls>
                     <ControlButton
                         disabled={nodes.length === 0}
-                        title={'Export viewport as PNG\nCtrl+Click to export to clipboard instead'}
+                        title={`Export viewport as PNG file\n\nHold ${
+                            isMac ? '⌥' : 'Ctrl'
+                        } to export to clipboard`}
```

Comment:
The new description makes it sound like simply holding <kbd>Ctrl</kbd>/<kbd>⌥</kbd> will export to clipboard. Could we bring back the "+Click"?

Reply:
done.